src/containers/sampleContainer.jsx

```diff
@@ -1,13 +1,12 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
-const SamplePresentationalComponent = ({ dispatch }) => {
-  return (
-    <div>
-      <p>Sub Component</p>
-    </div>
-  );
-};
+const SamplePresentationalComponent = () => (
```

Comment:
Hmm good question... I missed that before, but I think that's how it it was on master originally...